cli/cmd/info/info.go

```diff
@@ -24,6 +24,7 @@ Usage example:
 	dirctl info <digest>
 
 `,
+	SilenceUsage: true,
```

Comment:
Wouldn't it be better to set this in RootCmd and enable it where necessary?

Reply:
great point 👍 let me update! thanks for reviewing